internal/xds/translator/httpfilters.go

```diff
@@ -91,18 +91,20 @@ func newOrderedHTTPFilter(filter *hcmv3.HttpFilter) *OrderedHTTPFilter {
 	switch {
 	case filter.Name == wellknown.CORS:
 		order = 1
-	case filter.Name == basicAuthFilter:
+	case isFilterType(filter, basicAuthFilter):
 		order = 2
-	case isOAuth2Filter(filter):
+	case isFilterType(filter, oauth2Filter):
 		order = 3
 	case filter.Name == jwtAuthn:
 		order = 4
-	case filter.Name == wellknown.Fault:
+	case isFilterType(filter, extAuthFilter):
```

Comment:
should this be before `jwt` ?

Reply:
@arkodg you mean `jwtAuthn` should be `jwt`?

I just followed the Envoy filter name `envoy.filters.http.jwt_authn`

Reply:
@arkodg Is that because ext auth may add jwt token in the original headers?

Reply:
Yah